README.md

```diff
@@ -323,6 +322,109 @@ mode_backtrack
 
     ## [1] 1
 
+##### 3. Maximum Value After First Backtrack for the sequences
+
+Creating a function that returns maximum value after first backtrack.
+
+``` r
+gen_back_max <- function(n) {
+          ... 
```

Comment:
I might be a bit lazy to google but why do you put `...`? is it to make like a space? Just confused, that's all hehe